airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -69,22 +69,23 @@ def _parse_next_page_number(response: requests.Response) -> Optional[int]:
             return dict(parse_qsl(urlparse(next_page).query)).get("page")
         return None
 
-    def backoff_time(self, response: requests.Response) -> int:
+    def backoff_time(self, response: requests.Response) -> Union[int, float]:
         """
         The rate limit is 700 requests per minute
         # monitoring-your-request-activity
         See https://developer.zendesk.com/api-reference/ticketing/account-configuration/usage_limits/
         The response has a Retry-After header that tells you for how many seconds to wait before retrying.
         """
-        retry_after = response.headers.get("Retry-After")
-        if retry_after:
+
+        retry_after = int(response.headers.get("Retry-After", 0))
+        if retry_after and retry_after > 0:
             return int(retry_after)
         # the header X-Rate-Limit returns a amount of requests per minute
         # we try to wait twice as long
-        rate_limit = float(response.headers.get("X-Rate-Limit") or 0)
-        if rate_limit:
+
+        rate_limit = float(response.headers.get("X-Rate-Limit", 2))
+        if rate_limit and rate_limit > 0:
             return (60.0 / rate_limit) * 2
-        # default value if there is not any headers
         return 60
```

Comment:
I know this isn't a change from this PR but looks like you edited some stuff here so worth asking...

What's the reason for default rate limit being 1 request per minute? That seems especially slow

Reply:
I'm not sure that necessary headers are always exists. Ok, I've corrected this default value and it would return the super value.